### PR TITLE
Adds 2.65 Update and minor item fixes

### PR DESCRIPTION
**Pull request by Josemanki**
- Pandawa update
- Description changes on various classes
- Minor fixes on items and sets
- Various minor balance changes on Fogger and Ougi

### CODE REVIEW

File: server/app/suggester/trainer.py [hunk 1]
```diff
@@ -1,94 +1,94 @@
...
-    cache.set("suggestion_lookup_table_version", new_version)
-    cache.delete("suggestion_lookup_table:{}".format(old_version))
-
-    print("Done")
-
-
-if __name__ == "__main__":
-    args = parser.parse_args()
-    train_suggestion_model(get_relative_date(args.days))
+from collections import defaultdict
```
Comment (lightyearsaway) [inline server/app/suggester/trainer.py:1]: What are these changes?

File: server/oneoff/sync_class.py [hunk 2]
```diff
@@ -83,6 +83,7 @@ def add_buffs():
         with session_scope() as db_session:
             for class_name in all_classes:
                 for spell in all_classes[class_name]:
+                    print(spell["name"])
```
Comment (lightyearsaway) [inline server/oneoff/sync_class.py:86]: remove?

**Review by lightyearsaway — APPROVED** [review]
Does this list of commands look right?

sync_class
item -> black rat ring
item -> white rat sword
item -> clearing balgourde
set -> Punkin' Set

thank you for your contributions as always <3

File: server/app/suggester/trainer.py (hunk 1 above)
Comment (Josemanki) [reply server/app/suggester/trainer.py:1]: Honestly I have no idea. Have had those stuck forever and forgot to exclude them this time around 😓

Comment (Josemanki) [reply server/app/suggester/trainer.py:1]: It's the exact same file, so I have no idea haha.

**Comment by Josemanki** [comment]
> Does this list of commands look right?
> 
> sync_class
> item -> black rat ring
> item -> white rat sword
> item -> clearing balgourde
> set -> Punkin' Set
> 
> thank you for your contributions as always <3

Almost: You are also missing 
item -> black rat daggers

Otherwise looking fine!

**Pull request merged**